cmds/record.c

```diff
@@ -1373,10 +1369,14 @@ static void send_info_file(int sock, const char *dirname)
 	void *info;
 	int len;
 
-	xasprintf(&filename, "%s/info", dirname);
+	xasprintf(&filename, "%s/info.txt", dirname);
 	fd = open(filename, O_RDONLY);
-	if (fd < 0)
-		pr_err("open info failed");
+	if (fd < 0) {
+		xasprintf(&filename, "%s/info", dirname);
+		fd = open(filename, O_RDONLY);
+		if (fd < 0)
+			pr_err("open info failed");
+	}
```

Comment:
This kind of pattern is repeated many times over the entire source code. I think we might need a helper something like this and use it everywhere we open `info.txt` or `info` file.
```c
char *open_info_file(const char *dirname, int flags)
{
    char buf[PATH_MAX];
    struct stat statbuf;

    snprintf(buf, sizeof(buf), "%s/info.txt", dirname);
    if (stat(buf, &statbuf) < 0) {
        snprintf(buf, sizeof(buf), "%s/info", dirname);
        if (stat(buf, &statbuf) < 0)
            return -1;
    }
    return open(buf, flags);
}
```
We need it in the following parts as well.
https://github.com/namhyung/uftrace/blob/v0.12/cmds/recv.c#L479
https://github.com/namhyung/uftrace/blob/v0.12/cmds/recv.c#L287-L310

Because of this missing change, one of `recv` test fails as follows.
```
$ ./runtest.py -vdp -O0 167
Start 1 tests with 1 worker
Test case                 pg
------------------------: O0
build command: gcc -o t-sleep -fno-inline -fno-builtin -fno-ipa-cp -fno-omit-frame-pointer -D_FORTIFY_SOURCE=0  -pg -O0  s-sleep.c
prerun command: /home/honggyu/uftrace/uftrace recv --no-pager --no-event --libmcount-path=/home/honggyu/uftrace -d xxx --port 42877
prerun command: /home/honggyu/uftrace/uftrace record --no-pager --no-event --libmcount-path=/home/honggyu/uftrace --host localhost --port 42877 -E linux:schedule t-sleep
test command: /home/honggyu/uftrace/uftrace replay --no-pager  --libmcount-path=/home/honggyu/uftrace -d xxx/uftrace.data
uftrace: /home/honggyu/uftrace/utils/data-file.c:502:open_info_file
 ERROR: invalid magic string found!
 t167_recv_sched: -pg -O0 returns 1

 167 recv_sched          : NZ
```